qbft/spectest/tests/roundchange/duplicate_msg_quorum_prepared_rc_first.go

```diff
@@ -19,7 +19,8 @@ func DuplicateMsgQuorumPreparedRCFirst() *tests.MsgProcessingSpecTest {
 		testingutils.TestingPrepareMessage(ks.Shares[3], types.OperatorID(3)),
 	}
 	msgs := []*qbft.SignedMessage{
-		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[1], types.OperatorID(1), 2),
+		testingutils.TestingRoundChangeMessageWithRoundAndRC(ks.Shares[1], types.OperatorID(1), 2,
```

Comment:
why change?

Reply:
mistake in a previous PR (https://github.com/bloxapp/ssv-spec/pull/162/files#r1120058796)